src/App.js

```diff
@@ -441,12 +441,21 @@ function App() {
   }
 
   const copyToClipboard = () => {
-    navigator.clipboard.writeText("FOCLACH " + getTodaysWordNumber() + ' - ' + rowsPlayed + "/6\x0A"  + myResults + "\x0A").then(function(){
-      setClipboardMessage(dictionary['ResultsCopiedToClipboard'])
-      showMessage(dictionary['ResultsCopiedToClipboard'], { className: 'infoToast'})
-    }, function(){
-      console.log('there was a problem heuston')
-    });
+    if ((gameState === state.lost) {
```

Comment:
So might be able to avoid the repetition of the if else statement if you use a ternary operator instead. 

Something like this: 

let rowString = gameState === state.lost ? 'X' : rowsPlayed 

And then replace `rowsPlayed` with your new `rowString` that will be either X or the amount of rows played depending on whether the game was won or lost.